src/icons/icons.qrc

```diff
@@ -360,5 +360,7 @@
         <file>view-statistics.svg</file>
         <file>wallet-open.svg</file>
         <file alias="qbittorrent.svg">qbittorrent-tray.svg</file>
+        <file>rss_read_article.png</file>
+        <file>rss_unread_article.png</file>
```

Comment:
wrong order.

Reply:
QtCreator (with which I edited `icons.qrc`) doesn't give a damn about it...
Fixed.